src/hotspot/share/gc/g1/g1EvacFailureObjectsSet.hpp

```diff
@@ -74,7 +74,7 @@ class G1EvacFailureObjectsSet {
 
   // Apply the given ObjectClosure to all objects that failed evacuation. Objects
   // are passed in increasing address order.
-  void iterate(ObjectClosure* closure);
+  void process_and_drop(ObjectClosure* closure);
```

Comment:
The comment needs some update, something like: "Apply the given ObjectClosure to all objects that failed evacuation and empties the list after processing. Objects are passed in increasing address order."